handbook/ce/engaging-other-teams.md

```diff
@@ -1,5 +1,170 @@
 # Engaging Other Teams
 
-This is forthcoming (being worked on by [Virginia](https://about.sourcegraph.com/company/team#virginia-ulrich-she-her)) and is meant to consolidate/revise [CE-to-Engineering issue handover process](ce_to_eng_handover.md) and [How to route questions from customers](routing_questions.md). It will also touch on the fact that we never want CSEs to be gatekeepers and have a template for details CSEs will include when seeking help from engineering. And it will include details about engaging marketing, etc.
+Our goal as Customer Support is to only need help from engineering no more than 10% of the time (not counting when we file defects). It's okay that we need help, and we will need help, and we also continually take what we learn each time we do back to the rest of the team (and our product documentation, as relevant).
 
-Once drafted, Virginia will ask that all eng managers and CEs review
\ No newline at end of file
+When we need help, we seek it consistently, following the steps outlined here. We created these steps to ensure we:
+
+* Take the work as far as we can
+* Exhaust our collective expertise
+* Minimize the time added to the customer experience whilst they wait for help
+
+At no point are we meant to be gate keepers or go-betweens. Generally, when a CE needs help, they will come to us. They may also still go directly to engineering for other questions they have. And when we engage engineering, if the CE wants to be included, be sure to do so. This is especially relevant for customers still in the pre-sales process where everything is just a bit more sensitive. 
+
+## Step 1: Gather all the details/context and set expectations with the customer
+The more we know, the easier it will be to find the answer. For example:
+
+* Do we have updated details about the customer's environment?
+* Do we have the steps to be able to reproduce the issue and relevant screenshots and logs?
+* Do we know if the customer made any other changes to their environment recently (this could be anything, not just related to Sourcegraph)?
+* Do have a few examples of users who are impacted and those who are not If the issue is not impacting all of their users?
+* Do we have an understanding of the level of urgency the customer and/or our CE/Sales counterparts are feeling to resolve the issue?
```

Comment:
I know know later in the page you call out how we categorize urgency levels, perhaps we link to that here too?

Reply:
I will think on this suggestion. My rationale for not is that any context a customer and/or CE/sales rep may have about urgency is just one data point that can inform priority. I don't ever want that to be the only data point, so I wrote it as two separate sequences, if you will. Thinking, though...